src/app/[locale]/components/edit/[id]/components/EditComponent.tsx

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) TOSHIBA CORPORATION, 2023. Part of the SW360 Frontend Project.
+// Copyright (C) Toshiba Software Development (Vietnam) Co., Ltd., 2023. Part of the SW360 Frontend Project.
+
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+
+// SPDX-License-Identifier: EPL-2.0
+// License-Filename: LICENSE
+
+'use client'
+import { useEffect, useState, useCallback } from 'react'
+
+import CommonTabIds from '@/object-types/enums/CommonTabsIds'
+import ApiUtils from '@/utils/api/api.util'
+import { Session } from '@/object-types/Session'
+import HttpStatus from '@/object-types/enums/HttpStatus'
+import { signOut } from 'next-auth/react'
+import { notFound, useRouter } from 'next/navigation'
+import { useTranslations } from 'next-intl'
+import { COMMON_NAMESPACE } from '@/object-types/Constants'
+import { SideBar, PageButtonHeader } from '@/components/sw360'
+import { toast, TypeOptions, ToastContainer } from 'react-toastify'
```

Comment:
I like Toastify, but for the sake of consistency, can you look if we have something in react-bootstrap ?

Reply:
thanks, I will look into it and update it along with Release Function. After the PR is merged.